lib/checkPullRequestBranch.js

```diff
@@ -23,28 +22,88 @@ const checkBranch = async (context) => {
    */
   const pullRequest = context.payload.pull_request;
   const branchName = pullRequest.head.ref;
-  const branchIsInvalid = DISSALLOWED_BRANCH_PREFIXES.some((prefix) => {
-    return branchName.startsWith(prefix);
-  });
+  
 
-  if (branchIsInvalid) {
+  if(branchName === 'develop'){
     const prAuthor = pullRequest.user.login;
     // Comment on the pull request.
     const wiki = 'wiki'.link(
       'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
       'instructions-for-making-a-code-change'
     );
+    
     const commentParams = context.repo({
       issue_number: pullRequest.number,
       body:
         'Hi @' + prAuthor +
-        ', PRs made from develop branch or from a branch ' +
+        ', PRs made from develop branch are not allowed. Also branches ' +
         'whose name is prefixed with develop, release or test are not ' +
         'allowed. So this PR is being closed. Please make your changes in ' +
         'another branch and send in the PR. To learn more about contributing ' +
         'to Oppia, take a look at our ' +
         wiki + ' (Rule 1 specifically). Thanks!',
     });
```

Comment:
This can be refactored to store the message for each case in a messages variable, and then make the request to comment on the PR at the end.